### ISSUE TITLE
Keyboard: Context key opens context menus, but the menu items don’t work

### ISSUE THREAD

**Issue by Timwi** [gitextensions/gitextensions#8390]
## Steps to reproduce

* Put keyboard focus on an item in the left pane (e.g. the first remote under “Remotes”).
* Press the Context key.
    ![image](https://user-images.githubusercontent.com/1565951/89671813-eb97ee80-d8e3-11ea-98a3-f6aee0539ab2.png)
* Press either M (“Manage”) or use arrow keys and press Enter to select Manage.

## Current behaviour

The menu closes and nothing else happens.

## Expected behaviour

The selected menu item should be executed.

## Environment

- Git Extensions 3.4.3.9999
- Build d4b0f48bbf3e39a71f5d7ff5231be5678d4aa0f1
- Git 2.28.0.windows.1
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4180.0
- DPI 96dpi (no scaling)
